### PR TITLE
chore(release): Angular version upgraded

### PR DESCRIPTION
**Pull request by romanblanco**
[Angular 1.5. released](http://angularjs.blogspot.cz/2016/02/angular-150-ennoblement-facilitation.html)

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   ],
   "main": "./dist/angular-bootstrap-switch.js",
   "dependencies": {
-    "angular": "~1.4.0",
```
Comment (icholy) [inline bower.json]: @romanblanco shouldn't this be `~1.5` ??

Comment (romanblanco) [inline bower.json]: @icholy I don't think so.
By `~1.5` I'd have only enabled support for **1.5** and not for **1.4**.
By using `~1.4` I enable support for both the newest version (so **1.5** atm.) and also the older one (**1.4**).

Nice explaination [here](http://stackoverflow.com/a/20412378)

Comment (icholy) [inline bower.json]: @romanblanco ah ok.

**Comment by frapontillo** [comment]
You should adhere to the AngularJS guidelines for commit messages.

**Comment by romanblanco** [comment]
@frapontillo Thanks, should be better now.

**Comment by icholy** [comment]
@romanblanco I don't think that's right. 

See [this](https://github.com/frapontillo/angular-bootstrap-switch/commit/c0856132dfe39d3bd03e94116e25683ec95cc7b6) commit for an example.

`chore(npm): update Angular`

**Comment by frapontillo** [comment]
@icholy is right, you are not making a release as you are just updating dependencies.

**Comment by romanblanco** [comment]
@frapontillo @icholy Corrected. I hope it's OK now.

**Comment by frapontillo** [comment]
Merged in 873821867d1b0afe424631a5754a98edbefa2bcd.

**Pull request closed without merging**